recipe/recipe.yaml

```diff
@@ -0,0 +1,152 @@
+context:
+  version: "6_5_20240922"
+  pkg_version: ${{ version | split("_") | join(".") }}
+
+package:
+  name: ncurses
+  version: ${{ pkg_version }}
+
+source:
+  url: https://github.com/ThomasDickey/ncurses-snapshots/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: 75164543acb98d7b660d1a98359a7009b092551d49c60632c4f7976647960478
```

Comment:
Pretty sure we don't want to build the `_calver` snapshots (compare repo URL), only released versions.

Reply:
Ah OK! I got confused by Mr. AutotickBot. Reverted to 6_5 from current main.